.github/workflows/main.yml

```diff
@@ -16,7 +16,7 @@ env:
   # go needs absolute directories, using the $HOME variable doesn't work here.
   GOCACHE: /home/runner/work/go/pkg/build
   GOPATH: /home/runner/work/go
-  GO_VERSION: 1.18.4
+  GO_VERSION: 1.19.13
```

Comment:
Any reason to not go further to 1.21.6?

Reply:
I wasn't sure if there is a reason you had things pinned on an earlier version. I went with 1.19 to match what was used in lnd. But I can update it all the way to latest versions.

Reply:
No reason, good reason really, updating to the newer version should also give us some free perf improvements.